packages/samples/test/output/azure/resource-manager/arm-library/@azure-tools/typespec-autorest/openapi.json

```diff
@@ -1764,6 +1764,167 @@
         }
       }
     },
+    "Azure.ResourceManager.CommonTypes.ManagedServiceIdentityType": {
```

Comment:
is this expected to not use the common type anymore? is it because it should be a "patch" model with everything optional?

Reply:
forgot to delete